### PR TITLE
:sparkles: global search bar in category

### LINKED ISSUE #537: Globals search bar
Should be good that all search bar can find Skillz even if they don't belong to the actual category

As a user I would like find all the skills in the search bars in order to don't waste time to change page to find the skill searched

### PR DESCRIPTION
**Pull request by MailyLehoux**
resolve #537

### CODE REVIEW

**Comment by MailyLehoux** [comment]
will be weird to test : according to #553 we can see the results of all the skills that are already noted. But for the search I used  the skills list

File: src/components/organisms/AddSkilListSelector.tsx [hunk 1]
```diff
@@ -32,6 +40,9 @@ const AddSkillListSelector = ({
   const [openSkillDetails, setOpenSkillDetails] = useState(false);
   const [selectedSkill, setSelectedSkill] = useState(null);
   const tutorialModeValue = useContext(TutorialModeContext);
+  const [debouncedSearchValue] = useDebounce(search, 500);
```
Comment (benjaminbrgs) [inline src/components/organisms/AddSkilListSelector.tsx:43]: And then delete it

File: src/components/organisms/AddSkilListSelector.tsx [hunk 2]
```diff
@@ -42,6 +53,13 @@ const AddSkillListSelector = ({
     setOpenSkillDetails(true);
   };
 
+  /*
+   * QUERIES
+   */
+  const { data } = useQuery<SearchSkillsVerifiedQuery>(SEARCH_SKILLS_VERIFIED, {
+    variables: { search: `%${debouncedSearchValue}%` },
```
Comment (benjaminbrgs) [inline src/components/organisms/AddSkilListSelector.tsx:60]:
```suggestion
    variables: { search: `%${search}%` },
```
Here you can use directly the `search` prop as useDebounce is already used in the parent component.

File: src/graphql/queries/skills.ts [hunk 3]
```diff
@@ -95,6 +95,14 @@ export const SEARCH_SKILLS_AND_PROFILES_QUERY = gql`
   }
 `;
 
+export const SEARCH_SKILLS_VERIFIED = gql`
```
Comment (benjaminbrgs) [inline src/graphql/queries/skills.ts:98]: There's already the `GET_ALL_VERIFIED_SKILL` query with a search value, maybe we can use it ?

File: src/components/organisms/AddSkilListSelector.tsx (hunk 2 above)
Comment (benjaminbrgs) [reply src/components/organisms/AddSkilListSelector.tsx:60]: This also avoid a delay in the refresh of result number in other categories

**Review by benjaminbrgs — CHANGES_REQUESTED** [review]
I've suggested some changes but it works perfectly 👌🏼

**Review by benjaminbrgs — APPROVED** [review]

**Pull request merged**